microsoft/surface/firmware/ipts.nix

```diff
@@ -0,0 +1,13 @@
+{ stdenv, pkgs }:
+let
+  repos = (pkgs.callPackage ../repos.nix {});
+in
+stdenv.mkDerivation {
+  name = "microsoft-surface-ipts-firmware";
```

Comment:
Hi, linux-surface co-maintainer here. FYI, the ipts firmware files are only needed if you are using kernel 4.19 with the old IPTS driver. With kernel 5.4+ and the new driver these are not used anymore and dont need to be installed.

Reply:
In theory older kernels in NixOS but its probably a good idea to just settle on the newer ones for this hardware.

Reply:
> Hi, linux-surface co-maintainer here. FYI, the ipts firmware files are only needed if you are using kernel 4.19 with the old IPTS driver. With kernel 5.4+ and the new driver these are not used anymore and dont need to be installed.

Thanks for that -- I hadn't realised that had changed from the docs, I'm afraid.

I'll follow-up over there, except for asking whether that includes the Surface Go and SL3 variations?
As a few notes in either the `iptsd` or `intel-precise-touch` repos imply they're different?

Reply:
> In theory older kernels in NixOS but its probably a good idea to just settle on the newer ones for this hardware.

I agree.
But, I'm also tempted wrap it with an option switch rather than remove it entirely, though?

Reply:
> I'll follow-up over there, except for asking whether that includes the Surface Go and SL3 variations?
> As a few notes in either the `iptsd` or `intel-precise-touch` repos imply they're different?

The Surface Go line and the AMD version of the Surface Laptop 3 don't use IPTS. The touchscreen on the Go's is supported upstream, while the one on the SL3-AMD doesn't work at all right now.

Reply:
> > I'll follow-up over there, except for asking whether that includes the Surface Go and SL3 variations?
> > As a few notes in either the `iptsd` or `intel-precise-touch` repos imply they're different?
> 
> The Surface Go line and the AMD version of the Surface Laptop 3 don't use IPTS. The touchscreen on the Go's is supported upstream, while the one on the SL3-AMD doesn't work at all right now.
> 

Oh, grrr! I completely misunderstood that part, then!
Thanks very much!

Reply:
> Hi, linux-surface co-maintainer here. FYI, the ipts firmware files are only needed if you are using kernel 4.19 with the old IPTS driver. With kernel 5.4+ and the new driver these are not used anymore and dont need to be installed.

I've removed the derivative for the `ipts` firmware, thanks.